notebooks/scripts/make_mutation_table.py

```diff
@@ -0,0 +1,73 @@
+from augur.io import read_sequences
+import pandas as pd
+import argparse
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument("--alignment", required=True, help="the name of the aligned consensus file WITH reference")
+    parser.add_argument("--output", required=True, help="outputting a tsv file of mutations per cluster")
+
+    args = parser.parse_args()
+
+
+    sequences = read_sequences(args.alignment)
+
+    # The reference is the first sequence.
+    reference = next(sequences)
+
+    # The rest of the sequences are from the clusters.
+    clusters = list(sequences)
+
+    ignored_clusters = ["unassigned"]
+    mutations = []
+    for cluster in clusters:
+        if cluster.name in ignored_clusters:
+            continue
+
+        for site in range(len(reference.seq)):
+            if cluster.seq[site] != reference.seq[site]:
+                # Report the site in 1-based coordinates.
```

Comment:
Looking at specific results for MERS, I see now that we need to ignore more than just specific clusters in the pairwise comparisons:

 - specific characters like "X" that indicate a lack of consensus (and maybe even N, potentially, where the consensus is also uninformative)
 - leading gaps in the reference's aligned sequence that most likely represent lack of sequence for the reference instead of a biologically-meaningful insertion

Both of these classes of characters artificially inflate the list of mutations per cluster.

Reply:
1b4a675 resolves this.